rosbag2_cpp/include/rosbag2_cpp/clocks/player_clock.hpp

```diff
@@ -186,25 +117,13 @@ class PlayerClock
   ROSBAG2_CPP_PUBLIC
   virtual void jump(rclcpp::Time time) = 0;
 
-  /**
-   * \brief Add callbacks to be called when a time jump exceeds a threshold.
-   * \details Callbacks specified in JumpHandler object will be called in two cases:
-   *   1. use_sim_time is true: if the external time source jumps back in time, or forward
-   * farther than the threshold.
-   *   2. use_sim_time is false: if jump(time_point) is called and time jumps back or forward
-   * farther than the threshold.
-   * \param handler Shared pointer to the JumpHandler object returned from JumpHandler::create(..)
-   * \throws std::invalid argument if jump threshold has invalid value.
-   */
-  ROSBAG2_CPP_PUBLIC
-  virtual void add_jump_calbacks(PlayerClock::JumpHandler::SharedPtr handler) = 0;
-
-  /**
-   * \brief remove jump callbacks from processing list.
-   * \param handler Shared pointer to the JumpHandler object returned from JumpHandler::create(..)
-   */
+  /// Add a callback to invoke if the jump threshold is exceeded.
+  /// \sa rclcpp::Clock::create_jump_callback
   ROSBAG2_CPP_PUBLIC
-  virtual void remove_jump_callbacks(PlayerClock::JumpHandler::SharedPtr handler) = 0;
+  virtual rclcpp::JumpHandler::SharedPtr create_jump_callback(
```

Comment:
@emersonknapp If you are removing jump callbacks from player_clock class, why do you need this API `create_jump_callback(..)` which is returning `null_ptr`?

Reply:
I meant "it's an implementation that was mostly copy-pasted from rclcpp". I will use the jump handler in the ROStime implementation https://github.com/ros2/rosbag2/pull/928 - i could have added this there but it will end up the same